core/tests/protractor_utils/AdminPage.js

```diff
@@ -91,12 +91,11 @@ var AdminPage = function() {
         by.css('.protractor-test-reload-exploration-button')
       );
     };
-
     this.reloadCollection = async function(collectionId) {
       this.get();
```

Comment:
I'm not sure how this got missed, but we need an `await` here. I know this isn't related to your changes, but could you add it since you're already working on this file?

```suggestion
      await this.get();
```